manifests/plugin/openscap.pp

```diff
@@ -6,4 +6,10 @@
 #
 class foreman::plugin::openscap {
   foreman::plugin {'openscap': }
+
+  if $foreman::cli::manage_root_config {
```

Comment:
Sorry, it has been a while since I've written puppet code. Does that mean that I'd have to create a new puppet class `foreman::cli::plugin::openscap` which would include both foreman and foreman::cli classes and check `$foreman::cli::manage_root_config` and `$foreman::hammer_plugin_prefix` and let users to explicitly instantiate this class? If that's the case, is there any way to keep the logic here so the openscap plugin would automatically install the hammer plugin if the plugin is enabled and foreman::cli manages the config? Maybe using some `defined("foreman::cli")` and `require`/`include`?